include/lo2s/perf/event_provider.hpp

```diff
@@ -82,14 +45,14 @@ class EventProvider
         return instance_mutable();
     }
 
-    static EventDescription get_event_by_name(const std::string& name);
+    static SysfsEvent get_event_by_name(const std::string& name);
```

Comment:
SysfsEvent -> PerfEvent for everything besides get_pmu_events (which uses sysfs and thus can only return SysfsEvents)